.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,40 +1,27 @@
-<!--- Hi, and thanks for contributing! -->
-<!--- Make sure to provide a general summary of your changes in the title above. -->
-<!--- For example: "[scsynth] Fix crash when encountering cute kittens" -->
+<!-- Please see CONTRIBUTING.md for guidelines. -->
 
 Purpose and Motivation
 ----------------------
 
-<!--- Why is this change required? What problem does it solve? -->
-<!--- If it fixes an open issue, please link to it here by writing "Fixes #555". -->
+Fixes #
 
-Types of changes
-----------------
+Type of change
+--------------
 
-<!--- What types of changes does your pull request introduce? -->
-<!--- Some examples are below (you can delete the lines that don't apply): -->
+<!-- Delete lines that don't apply -->
 
-- Documentation (non-code change which corrects or adds documentation for existing features)
-- Bug fix (non-breaking change which fixes an issue)
-- New feature (non-breaking change which adds functionality)
-- Breaking change (fix or feature that would cause existing functionality to change)
+- Documentation
+- Bug fix
+- New feature
+- Breaking change
 
-Checklist
----------
+To-do list
+----------
 
-<!--- Complete an item by checking it: [x] -->
+<!-- Complete an item by checking it: [x] -->
+<!-- Delete lines that don't apply, add new entries to track your progress -->
```

Comment:
I am strongly against people deleting lines from this checklist. Having all the items there every time makes contributing more conscientious. I have seen people delete lines for tasks that obviously should have been a part of the PR.